src/stub.c

```diff
@@ -374,8 +376,12 @@ int CALLBACK _tWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPTSTR lpCm
    return 0;
 }
 
+/* The NTHeader is another name for the PE header. It is the 'modern' executable header
+   as opposed to the DOS_HEADER which exists for legacy reasons */
 static PIMAGE_NT_HEADERS retrieveNTHeader(LPVOID ptr) {
   PIMAGE_DOS_HEADER dosHeader = (PIMAGE_DOS_HEADER)ptr;
+
+  /* e_lfanew points to the NTHeader (aka PE Header) */
```

Comment:
You mean "is the offset to"?

Reply:
@jakedouglas ah yeah you're right. The total expression is the pointer, but just `e_lfanew` is an offset, once its added to the `dosHeader` it becomes a pointer to the `PE header` though..thanks 👍

Reply:
👍